media/css/base/banners/turning-red.scss

```diff
@@ -0,0 +1,201 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+$brand-theme: 'firefox';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+$color-panda-red: #fc4055;
+
+#turning-red-banner {
+    &.c-banner {
+        @include clearfix;
+        @include at2x('/media/img/banners/turningred/mobile.jpg', 320px, 110px);
```

Comment:
Change this URL to `https://www.allizom.org/media/img/banners/turningred/mobile.jpg` for local testing.

Reply:
Suggestion (non-blocking): to save confusion later, should we update these images to use the absolute prod paths, along with a comment explaining why they are not in the source repo?